pallets/rmrk-core/src/functions.rs

```diff
@@ -461,7 +478,10 @@ where
 		NFTs::<T>::remove(collection_id, nft_id);
 		let kids = Children::<T>::take((collection_id, nft_id));
 		for (child_collection_id, child_nft_id) in kids {
-			Pallet::<T>::recursive_burn(child_collection_id, child_nft_id, max_recursions - 1)?;
+			match Pallet::<T>::recursive_burn(child_collection_id, child_nft_id, max_recursions - 1) {
```

Comment:
This is the same as the old code. Changes are not needed. It's ok to just document the caller function must be transactional